docs/rules/block-spacing.md

```diff
@@ -0,0 +1,59 @@
+# Disallow or enforce spaces inside of single line blocks. (block-spacing)
+
+This rule is for spacing style about single line blocks.
```

Comment:
about -> around. Or probably even "within", since this rule is controlling spacing inside single line blocks.